WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreStates.kt

```diff
@@ -103,10 +104,15 @@ sealed class RestoreRequestState {
         val rewindId: String,
         val progress: Int?,
         val message: String? = null,
-        val currentEntry: String? = null
+        val currentEntry: String? = null,
+        val date: Date? = null
```

Comment:
Your are totally right! 🌟 

I am going to rename that to `published` and match that with the `ActivityLogModel`, since this is where that date comes from in the first place. Wdyt?

Reply:
This is done here: 4b5a9fda72cf636abcd3864be970fb1f02de7624

PS: Let me know if you want it renamed to something else. 🙏

Reply:
Awesome, thanks!